core/drivers/firewall/stm32_etzpc.c

```diff
@@ -442,8 +442,15 @@ static TEE_Result stm32_etzpc_configure(struct firewall_query *firewall)
 		attr = etzpc_binding2decprot(mode);
 
 		if (decprot_is_locked(id)) {
-			EMSG("Peripheral configuration locked");
-			return TEE_ERROR_ACCESS_DENIED;
+			if (etzpc_get_decprot(id) != attr) {
+				EMSG("Peripheral configuration locked");
+				return TEE_ERROR_ACCESS_DENIED;
+			}
+
+			DMSG("Valid access for periph %"PRIu32" - attr %s",
```

Comment:
Maybe something like "Same configuration is already set and locked"

Reply:
I'll go for `DMSG("Compliant locked config for periph %"PRIu32" - attr %s",`.
Fine to you?